src/main/java/com/minecraftabnormals/abnormals_core/core/util/DataUtil.java

```diff
@@ -173,23 +179,33 @@ public static boolean matchesKeys(ResourceLocation loc, RegistryKey<?>... keys)
 				return true;
 		return false;
 	}
-
+	
 	/**
-	 * <p>Registers a {@link IDispenseItemBehavior} that will perform the new behavior if the condition is met and the behavior that was already in the registry if not.
-	 * This works even if multiple mods add new behavior to the same item.</p>
-	 * <p>Ideally, the condition should be implemented such that the predicate only passes if the new behavior will be 'successful', avoiding problems with failure sounds not playing.</p>
+	 * <p>Slates a {@link AlternativeDispenseBehavior} instance for later processing, where it will be used to register
+	 * an {@link IDispenseItemBehavior} that performs the new behavior if its condition is met and the behavior that was 
+	 * already registered if not. See {@link AlternativeDispenseBehavior} for details.
+	 *
+	 * <p>Since Abnormals Core handles registering the condition at the right time, mods should call this method as early as possible,
+	 * such as in a {@code static{}} block.</p>
```

Comment:
Statically registering here would not be a good practice because it's possible for item and block registries to not be done yet.
```suggestion
	 * <p>Since Abnormals Core handles registering the condition at the right time, mods should call this method as early as possible.</p>
```